src/validateInvocation.ts

```diff
@@ -16,22 +16,20 @@ export default async function validateInvocation(
     'Validating integration config...',
   );
 
-  if (await isConfigurationValid(context.instance.config)) {
-    context.logger.info(authenticationSucceededMessage);
-  } else {
-    throw new Error(authenticationFailedMessage);
+  if (!context.instance.config.apiKey) {
+    throw new IntegrationValidationError(
+      'ERROR: Context is missing apiKey configuration variable',
+    );
   }
-}
-
-async function isConfigurationValid(config: {
-  apiKey: string;
-}): Promise<boolean> {
-  if (!config.apiKey) return false;
 
   try {
-    await requestAll('/users', 'users', config.apiKey, 1);
-    return true;
+    await requestAll('/users', 'users', context.instance.config.apiKey, 1);
   } catch (e) {
```

Comment:
Previously this suppressed all errors and _always_ threw 'Failed to authenticate with given apiKey'.